src/app/backend/scaling/scale.go

```diff
@@ -17,6 +17,7 @@ package scaling
 import (
 	"strconv"
 
+	"k8s.io/api/apps/v1"
```

Comment:
Please add "v1" at the first of this line as package alias.
We need to name for packages that are just version number like "v1"  explicitly.
Please see https://github.com/golang/go/issues/29041#issuecomment-443285973